tests/Unit/DehydratePropertyFromWithFileUploadsTest.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Tests\Unit;
+
+use Illuminate\Support\Facades\Storage;
+use Livewire\Features\SupportFileUploads;
+use Illuminate\Http\UploadedFile;
+use Livewire\TemporaryUploadedFile;
+use Livewire\Wireable;
+
+class DehydratePropertyFromWithFileUploadsTest extends TestCase
+{
+    /** @test */
+    public function a_text_variable_should_return_with_no_changes()
+    {
+        if (version_compare(PHP_VERSION, '7.4', '<')) {
```

Comment:
@some-daan and @stevenmond how come each of these tests have a version compare on them?

To me it looks like you are testing the `SupportFileUploads` class directly, which doesn't have any types properties in it.

Can I suggest removing this from all of these tests? And we can then run tests again and see if anything breaks.

Reply:
@joshhanley Removed it from my PR. I copy-pasted from another test with it when I was originally doing the tests.

Reply:
@stevenmond thanks! Ran your tests and they all passed.

@some-daan if you could remove the version checks, that would be great 🙂

Reply:
@joshhanley I've removed the version checks

Reply:
@some-daan thanks!